examples/opentelemetry-otlp.graphql

```diff
@@ -0,0 +1,36 @@
+schema
+  @server(port: 8000, graphiql: true, hostname: "0.0.0.0")
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", httpCache: true)
+  @opentelemetry(
+    export: {
+      otlp: {
+        url: "https://api.honeycomb.io:443"
+        # gather api key from https://ui.honeycomb.io and set it as env when running tailcall
+        headers: [{key: "x-honeycomb-team", value: "{{env.HONEYCOMB_API_KEY}}"}]
```

Comment:
on honeycomb it looks like this 

![image](https://github.com/tailcallhq/tailcall/assets/8974488/88cb144c-d6cf-4e04-933f-9f5662e742df)